airflow/upgrade/rules/postgres_mysql_sqlite_version_upgrade_check.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.configuration import conf
+from airflow.upgrade.rules.base_rule import BaseRule
+from airflow.utils.db import provide_session
+
+
+class DatabaseVersionCheckRule(BaseRule):
+
+    title = "Check versions of PostgreSQL, MySQL, and SQLite to ease upgrade to Airflow 2.0"
+
+    description = """\
+From Airflow 2.0, the following database versions are supported:
+PostgreSQl - 9.6, 10, 11, 12, 13;
+MySQL - 5.7, 8;
+SQLite - 3.15+
+    """
+
+    @provide_session
+    def check(self, session=None):
+
+        more_info = "See link below for more details: https://github.com/apache/airflow#requirements"
+
+        conn_str = conf.get(section="core", key="sql_alchemy_conn")
+
+        if "sqlite" in conn_str:
+            result = session.execute('select sqlite_version();').fetchone()[0]
```

Comment:
```suggestion
            result = session.execute('select sqlite_version();').scalar()
```

Would be much cleaner I think:

```python
In [4]: session.execute('select sqlite_version();').fetchone()
Out[4]: ('3.30.0',)

In [5]: session.execute('select sqlite_version();').scalar()
Out[5]: '3.30.0'
```

Reply:
I agree!